css/pages/discord.css

```diff
@@ -1,9 +1,39 @@
+.main-heading{
+  text-align: left;
+  font-weight: 600;
+  padding: 10px;
+}
 body{
- font-family: Arial, Helvetica, sans-serif;
- font-weight: 500;
- padding-left: 20px;
+    margin-left: 5%;
+}
+
+.discord-info{
+  text-align: left;
+  font-weight: 500;
+  margin-left: 2%;
+
+}
+.get-started{
+  text-align: left;
+  font-weight: 550;
+  padding: 10px;
+}
+
+ul{
+    text-align: left;
+    font-weight: 500;
+    margin-left: 2%;
+    padding:10px
```

Comment:
; missing.
To avoid these formatting error, please install prettier VS code extesion and enable formatting on save. This will auto resolve all the formatting errors.